02-oop-inheritance/javaScript/refactored_JS/downloableMovie v0.1.js

```diff
@@ -0,0 +1,87 @@
+var movieObserver = function ( ) {
+  this.playing = function (title) {
+    console.log('A movie with title ' + title + ' has been played');
+  } 
+  this.stopped = function( title ) {
+    console.log('A movie with title ' + title + ' stopped');
+  } 
+}//end movieObserver
+
+var movie = function ( paramName, paramValue ) {
+  //Variables
+  this.attributes = [];
+  this.attributes[paramName] = paramValue; 
```

Comment:
The idea is to work with this as in a hashTable, or with a JSON object, where you set a pair of key, value in order to set the properties you want. 

E.G(as in line 92):  downMovie.set('duration','6 hours');

key : 'duration'
value:'6 hours'


This approach is different to the one used in: propertyName = value; 

But I choose the way I did it, so as to follow the model in the excercise. (Use a hashTable)